x-pack/plugins/canvas/public/expression_types/function_form.tsx

```diff
@@ -96,7 +96,7 @@ export class FunctionForm extends BaseForm {
         key: `${id}.${arg.name}.${valueIndex}`,
         ...passedProps,
         valueIndex,
-        onValueChange: onValueChange(arg.name, valueIndex),
+        onValueChange: debounce(onValueChange(arg.name, valueIndex), 100),
```

Comment:
I would recommend removing this change from this PR, since this PR is getting backported to 8.9.1 and is not related to this fix